components/brave_ads/common/notification_ads_features.cc

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/common/notification_ads_features.h"
+
+namespace brave_ads::features {
+
+namespace {}  // namespace
+
+BASE_FEATURE(kAdNotifications,
+             "AdNotifications",
```

Comment:
nit: name does not match feature

Reply:
it matches: `kAdNotifications` and `AdNotifications`.

Reply:
How about we deprecate this feature and move to `components/brave_ads/core/internal/ads/notification_ad_feature.h` as the timeout is related to that feature? Thanks

Reply:
done